.vale/styles/RedHat/DoNotUseTerms.yml

```
@@ -16,6 +16,7 @@ swap:
   debuggable: 'Do not use "debuggable". Rephrase the sentence to use the verb or noun debug. For example, change rebuild the debuggable version to rebuild the version that can be debugged.'
   foo: 'Do not use "foo". This term is technical jargon in code and as shorthand for fubar, an acronym of profanity in code.'
   fubar: 'Do not use "fubar". This term is an acronym of a profanity that is sometimes used by developers in code.'
+  green|yellow|blue: 'Do not use visual language, like color, to describe items in the us unless it is also described non-visually, like with the name of the item.' 
```

```suggestion
  \bred\b(?!-? *hat)|orange|yellow|green|blue|indigo|violet: 'Do not use colors to describe something unless it is also described non-visually, for example, with the name of the item.'
```

I tested this regex and it won't catch "redhat", "red hat" or "red-hat". 

Also added ROYGBIV colors because why not - unless there is a reason to only flag green/yellow/blue?

---

Looks good otherwise! Will be good to get this in